zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1852,16 +1852,19 @@ components:
             - ROLE
         permissions:
           type: array
+          description: The permissions to add/remove.
           items:
             properties:
               permissionType:
+                description: Specifies the type of permissions.
                 enum:
                   - CREATE
                   - READ
                   - UPDATE
                   - DELETE
               resourceIds:
                 type: array
+                description: A list of resource ids the permission relates to.
```

```suggestion
                description: A list of resource IDs the permission relates to.
```

💭 I can't find anything in our writing style guide, but @camunda/tech-writers it seems to me that we probably have a standard for the plural form of "IDs"?

---

I actually capitalized it first and removed it due to this part of the guidelines:

Follow these rules for casing:
Use "Sentence case" (with no period) for path tags properties.
Use "Sentence case" (with no period) for path summary properties.
Use "Sentence case." (with a period) for all other descriptive text.
Use "lower case" to reference API resources in descriptive text.
Example: use "The decision definition search query failed." instead of "The Decision Definition Search Query failed."

---

ah, thank you for mentioning that! Let's see what the technical writers think about this, and then I'll work with them to update that guidance to be more clear.

---

My preference here would be `IDs`, but I don't think we have a consensus on this yet (it's currently [up for discussion](https://github.com/camunda/developer-experience/issues/197)). As long as we're consistent within this PR/in general for the spec changes, I think it should be fine.

---

I have changed it to `IDs` as in other parts of the spec we also use `ID` 🙂